test/unit/testUtil.js

```diff
@@ -101,19 +101,19 @@ define('testUtil', ['bluebird', 'json!/test/testConfig.json'], (Promise, TestCon
      */
     function waitForElement(documentElement, selector, doThisFirst) {
         return new Promise((resolve) => {
-            const element = document.querySelector(selector);
+            const element = documentElement.querySelector(selector);
```

Comment:
Found a little bug here - we should be looking under the given `documentElement`, and include the `subtree` in the observer, right?

Reply:
IIRC this was supposed to be elements appearing as a direct child of `documentElement`, but it doesn't do any harm to expand it to be anything with `documentElement` as an ancestor. 👍

Reply:
I used it for a test where I have a handle on the parent element, but the child that I'm waiting on would be nested somewhere pretty deep, though I did have an tag and class to select for it.